docs/book/src/quick-start.md

```diff
@@ -7,9 +7,17 @@ This Quick Start guide will cover:
 - [Running locally](#test-it-out-locally)
 - [Running in-cluster](#run-it-on-the-cluster)
 
+## Prerequisites
+
+- [go](https://golang.org/dl/) version v1.12+.
+- [docker](https://docs.docker.com/install/) version 17.03+.
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) version v1.11.3+.
+- [kustomize](https://sigs.k8s.io/kustomize/docs/INSTALL.md) v3.1.0+
+- Access to a Kubernetes v1.11.3+ cluster.
+
 ## Installation
 
-Install [kubebuilder](https://sigs.k8s.io/kubebuilder):
+Following the steps to install [kubebuilder](https://sigs.k8s.io/kubebuilder):
```

Comment:
This is only part of a sentence now, and doesn't really make sense in isolation.  `Install` or `Follow` is an imperative -- it gives someone a command.  This form states what you're doing, and generally in this structure you'd expect it to be followed by another clause (e.g. `Following the steps to install KubeBuilder, Agnes ran go get`, `Agnes installed kubebuilder by following the steps below`).  Alternative, the phrase `The following` is short for `The following <plural noun>`, but you can't leave off the `The`.

Please switch back to an imperative form.

Reply:
Tks English is not my first language and this explanation was very helpful,